Utils/Dataflow/069_upload2es/load_data.sh

```diff
@@ -19,14 +30,26 @@ ES_CONFIG="${base_dir}/../../Elasticsearch/config/es"
 
 while [ -n "$1" ]; do
   case "$1" in
-    --config|-c)
+    -c|--config)
       [ -n "$2" ] && ES_CONFIG="$2" || { usage >&2 && exit 1; }
       shift;;
+    -h|--help)
+      usage
+      exit;;
+    -E|--eop)
+      EOP="$2"
+      shift;;
+    -B|--eob)
+      EOB="$2"
+      shift;;
+    -b|--batch)
+      BATCHMODE="${2,,}"
+      shift;;
     --)
       shift
       break;;
     -*)
-      echo "Unknown option: $1" >&2
+      log ERROR "Unknown option: $key" $(usage)
```

Comment:
1. This change has nothing to do with custom EOB/EOP.
2. `$key` variable is not defined.